integration_files/SweRV_EH1/riscv_dv_extension/testlist.yaml

```diff
@@ -47,14 +47,6 @@
   rtl_test: core_base_test_benchmark
   no_iss: 1
   no_post_compare: 1
-  
```

Comment:
Why did we remove these?

Reply:
The c_sample test was empty and wasn't being used. So removed it from the testlist.